elements/lisk-chain/src/block_assets.ts

```diff
@@ -46,6 +54,10 @@ export class BlockAssets {
 		return this._assets.find(a => a.moduleID === moduleID)?.data;
 	}
 
+	public getAllAssets(): BlockAsset[] {
```

Comment:
```suggestion
	public getAll(): BlockAsset[] {
```
I  think it's better not to repeat `assets.getAll()`